wavefront_cli/lib/proxy.py

```diff
@@ -40,6 +40,15 @@ def get_proxy_install_cmd(proxy_next):
         cmd = "curl -s %s | bash" % (pkg)
         cmd += " && apt-get -y -q install wavefront-proxy"
         return cmd
+    elif dist.strip() == "openSUSE":
+
+        pkg = proxy_pkg_rpm
```

Comment:
For this to work, your other PR (https://github.com/wavefrontHQ/java/pull/240) to be merged right?

Once that is merged, we may want to upload a new rpm to proxy-next and test this out. Guess we'd also have to push these changes into PyPi. I'll check with Conor on that, so that we can do a end-end test of everything.

Reply:
Exactly, To make end to end test  we need to push PR (wavefrontHQ/java#240) and create new rpm.